src/main/java/com/javagirls/MangoRoom/controller/ReservationController.java

```diff
@@ -37,4 +40,10 @@ public Reservation addReservation(@RequestBody ReservationDto reservation) {
         return service.findAllWithTime(time);
     }
 
+    @GetMapping("/rooms/available")
+    public List<RoomDto> getAvailableRooms(@RequestParam(required = false) LocalDateTime startDate,
+                                           @RequestParam(required = false) LocalDateTime endDate){
+        return service.getFreeRooms(startDate, endDate);
+    }
+
```

Comment:
Haniu, jeśli pole oznaczasz jako (required = false), trzeba pomyśleć o czymś, co będzie zwracane w przypadku jeśli użytkownik nic nie poda, bo w tej chwili występuje błąd serwera 500. Może np. wszystkie wolne pokoje na dzień dzisiejszy lub najbliższy tydzień?

Reply:
usunęłam required=false :D, bo nie wiem, jak to zrobić